packages/menu/src/MenuGroup.ts

```diff
@@ -32,6 +34,9 @@ export class MenuGroup extends SpectrumElement {
         return [menuGroupStyles];
     }
 
+    @property({ type: String, reflect: true })
+    public selects: undefined | 'single' | 'multiple';
```

Comment:
At least menu group needs a `selects="none"` option, as the default is really "not directly managed, but a parent element can manage". What I was curious about though is whether I should also add that to `sp-menu`.

In other words, at least menu group needs to distinguish between "my parent menu can manage my selection" and "the item should not not be selectable". This is a nuanced case I missed in the issue description so I'll update with what we decide here. My gut says we might as well add it to menu, since it's simpler to manage the overall logic and won't hurt anything.

Reply:
I think this makes sense. However, this might interest with the way the the `[selects]` selector is used to apply sizing rules from Spectrum CSS.

Reply:
In this way, maybe it's not `selects="none"` but `selects="inherit"` or similar and then no value for `selects` is the never select option?

Reply:
I'd definitely want menu group to default to `inherit`. I think the API would be pretty awkward otherwise.

So maybe that's specific to `menu-group`?

Reply:
I was thinking on this in the realm of styling. There is specific styling for the menu items when a selection can be made. If `select="none"` is the attribute to removes selection, but sometimes selection can happen when no attribute at all, then you add complexity to what the CSS rules should be processed to in that case. None of this is a blocker, just complexity to keep in mind.

Reply:
I think this would work, and as I'm thinking about it more I think the API different here is okay (but also something that should get explicitly called out in the `sp-menu-group` documentation). Since sp-menu-group manages the underlying sp-menu, I think this would avoid some of the css complexity you've mentioned.

**sp-menu** `selects: undefined | 'inherit' | 'single' | 'multiple' // default to not selectable
**sp-menu-group** `selects: undefined | 'none' | 'single' | 'multiple' // default to 'inherit' in the underlying sp-menu